components/ctas/Cta.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <a :class="{ button: true, 'button--secondary': secondary }" :href="to">
+  <component
+    :is="tag"
```

Comment:
This decision exposes an implementation detail giving too much power to the user and breaking component isolation. There are several forms of solving this:
- One is to emulate HTML `input` tag and provide some property collecting the intent: is this a `external="true"` CTA? If so, use `a`; else, use `nuxt-link`.
- Another is to make this intent explicit in the component name: `NavigationCta` (for internal navigation) vs `LinkCta` (for external navigation)
- A third one is to switch between implementations depending on the `to` property, distinguishing if it is a external or internal navigation in a similar way `isExternal` is used in the current component.

Other solutions are welcome as soon as they do not leak an implementation detail, do not expose more functionality than needed, and present a regular API, independent of the implementation (so, no separated `href` and `to` properties but only one).

Reply:
I have just upload a new commit with this fixed. If you think that the logic on the templated should be as `computed` properties, let me know. Since I though it's a very basic logic, it may not needed computed properties. Anyways, I hope it fixes this issue :)